src/domain/Popovers/DropdownMenu/subcomponents/style.ts

```diff
@@ -230,16 +234,32 @@ const StyledSection = styled.li`
   }
 `
 
-const StyledSectionContent = styled.div`
+const StyledSectionContentWithTooltip = styled.div`
   display: flex;
   justify-content: space-between;
 `
 
+const StyledSectionContent = styled.div`
+  display: flex;
+  align-items: center;
+  ${(props: IStyledSectionContent) => props.hasSecondaryText && css`
+    margin: -9px 0;
```

Comment:
Instead of the negative margin, I think we can just change the `padding` on line `189` from `1em` to `7px 1em` when `hasSecondaryText` is true which should keep everything the same height and the placement of the section content wont be hacky.